src/operator/softmax_output-inl.h

```diff
@@ -88,7 +92,7 @@ class SoftmaxOutputOp : public Operator {
       Tensor<xpu, 2> label = in_data[softmaxout_enum::kLabel].FlatTo2D<xpu, real_t>(s);
       Tensor<xpu, 3> out = out_data[softmaxout_enum::kOut].get_with_shape<xpu, 3, real_t>(s3, s);
       Tensor<xpu, 3> grad = in_grad[softmaxout_enum::kData].get_with_shape<xpu, 3, real_t>(s3, s);
-      SoftmaxGrad(grad, out, label);
+      SoftmaxGrad(grad, out, label, static_cast<real_t>(param_.ignore_label));
```

Comment:
Maybe I am missing something, but shouldn't this be accompanied by a change to mshadow?